support-frontend/assets/pages/contributions-landing/components/ContributionForm.jsx

```diff
@@ -116,7 +116,9 @@ const mapStateToProps = (state: State) => ({
   country: state.common.internationalisation.countryId,
   stripeV3HasLoaded: state.page.form.stripeV3HasLoaded,
   amazonPayOrderReferenceId: state.page.form.amazonPayData.orderReferenceId,
-  isRecaptchaPresentTest: state.common.abParticipations.recaptchaPresenceTest === 'recaptchaPresent',
+  isRecaptchaRequired: window.guardian.recaptchaV2 &&
+    (state.common.abParticipations.recaptchaPresenceTest === 'recaptchaPresent' ||
+    state.common.internationalisation.countryGroupId === 'AUDCountries'),
```

Comment:
I could have added the `isRecaptchaRequired` to the redux store (via a dispatch in `contributionsLandingInit.js`) but as this is all experimental and an A/B test it doesn't make sense right now. I can, if we think it's a better approach?